android/src/app/organicmaps/util/Utils.java

```diff
@@ -814,13 +814,6 @@ private static <T> T getParcelableOld(Bundle args, String key)
     return (T) args.getParcelable(key);
   }
 
-  @Nullable
-  public static <T> T getParcelable(@NonNull Intent intent, String key, Class<T> clazz)
-  {
-    final Bundle args = intent.getExtras();
-    return (args == null) ? null : getParcelable(args, key, clazz);
-  }
-
   @Nullable
   public static <T> T getParcelable(@NonNull Bundle args, String key, Class<T> clazz)
```

Comment:
According to the logic, something is wrong with these functions: getParcelable, getSerializable.
Remove them too?

Reply:
My logic here:
- You replaced removed getParcelable(Intent, ...) call as bad
- getParcelable(Intent, ...) calls getExtras and getParcelable(Bundle, ...) inside
Is it mean that getParcelable(Bundle, ...) also bad, no?